tg_odesli_bot/bot.py

```diff
@@ -375,8 +375,9 @@ def _compose_reply(
         """
         # Quote the original message for group chats
         if message.chat.type != ChatType.PRIVATE:
+            user_mention = message.from_user.get_mention(as_html=True)
             reply_list = [
```

Comment:
Show be one-liner: `reply_list = [f'<b>{user_mention} wrote:</b> {message_text}']`